### ISSUE TITLE
Infinite scroll broken w/ ember-in-viewport 3.5.0+

### ISSUE THREAD

**Issue by nickschot** [nickschot/ember-model-select#69]
Current workaround: set a resolution for ember-in-viewport 3.4.0.

**Comment by nickschot** [comment]
Did a patch release to work around this issue for now. See #70

**Comment by amauryt** [comment]
Hi Nick. Thank you for releasing this add-on and the related companion for `ember-bootstrap`.

I just wanted to give you a heads up and also to help other people that may have the same problem I had when upgrading to ember 3.11, as my model-selects stopped working with the following error:

```
Assertion Failed: You must call `this._super(...arguments);` when overriding `init` on a framework object. Please update <suoniamo-frontend@service:-observer-admin::ember463> to call `this._super(...arguments);` from `init`.
```
The origin is within `ember-in-viewport` and it is a [known issue](https://github.com/DockYard/ember-in-viewport/issues/204) in older releases of that add-on. However, I noticed that in #70 you locked the version of `ember-infinity`(and thus of `ember-in-viewport`) as a workaround for issue #69.

For the moment my workaround to is to completely disable `infiniteScroll` by setting it to false in model-selects options. Fortunately, for my application infinite scrolling is not absolutely necessary. Nonetheless, I think this that this workaround is far from ideal because the locked dependency is included even if it is not used, and it may cause conflicts with other add-ons or apps that use newer versions of ember or of the other add-ons.

**Comment by nickschot** [comment]
Thanks for the thorough heads up! I'll take a look to see if I can find the issue with ember-infinity (or if it might be resolved now).

**Comment by nickschot** [comment]
@amauryt seems this issue is solved in newer ember-in-viewport versions. I'm preparing an update!

**Comment by nickschot** [comment]
@amauryt should be solved in 0.2.0!

**Comment by amauryt** [comment]
> @amauryt should be solved in 0.2.0!

That was a quick response, thanks!
